checkov/common/checks_infra/solvers/attribute_solvers/jsonpath_not_equals_attribute_solver.py

```diff
@@ -0,0 +1,22 @@
+from typing import List, Optional, Any, Dict
+
+from checkov.common.checks_infra.solvers.attribute_solvers.jsonpath_equals_attribute_solver import JsonpathEqualsAttributeSolver
+from checkov.common.graph.checks_infra.enums import Operators
+
+
+class JsonpathNotEqualsAttributeSolver(JsonpathEqualsAttributeSolver):
+    operator = Operators.JSONPATH_NOT_EQUALS
+
+    def __init__(self, resource_types: List[str], attribute: Optional[str], value: Any) -> None:
+        super().__init__(resource_types=resource_types, attribute=attribute, value=value)
+
+    def get_operation(self, vertex: Dict[str, Any]) -> bool:
+        if self.attribute:
+            attribute_matches = self._get_attribute_matches(vertex)
+            if len(attribute_matches) == 0:  # the jsonpath is not found, so the check passes
+                return True
+
+        return super().get_operation(vertex)
```

Comment:
does it make sense to invoke `get_operation` of the parent class or wouldn't it be easier to just return `False`?

Reply:
But this is a little clunky. I will revisit it.

Reply:
got it